public/test/plugin.test.ts

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
should we close this one since it is carrying over all the other diffs

Reply:
I think we can close it. It is only carrying a PR that I didn't mean to backport to 2.16 after rebasing.